runner.py

```diff
@@ -1,138 +1,228 @@
-    parser.add_argument("-l", "--log-level",
-                        default='info',
-                        choices=logging_level,
-                        help="Specify verbosity level. By default uses level info.")
-
-    parser.add_argument("-s", "--serial",
-                        help="Specify serial to communicate with device board. "
-                             "Default value depends on the target")
-
-    parser.add_argument("-b", "--baudrate",
-                        default=config.DEVICE_SERIAL_BAUDRATE,
-                        help="Specify the connection speed of serial. "
-                             "By default uses 115200")
-
-    parser.add_argument("--no-flash",
-                        default=False, action='store_true',
-                        help="Board will not be flashed by runner.")
-
-    # Add alternative option "--long-test" "" for long-test argument, which is False
+    parser.add_argument(
+        "-T",
+        "--target",
+        default="ia32-generic-qemu",
+        choices=targets.keys(),
+        help="Set target on which tests will be run. "
+        "By default runs tests on %(default)s target. "
+        "Targets can be extended using extension feature.",
+    )
+
+    parser.add_argument(
+        "-H",
+        "--host",
+        default="pc",
+        choices=hosts.keys(),
+        help="Set host that run tests. \"pc\" host is recommended for running tests on emulators. "
+        "To control physical boards you can use \"rpi\" host or pc "
+        "(but it requires restarting board manually by yourself). By default runs tests on %(default)s. "
+        "Hosts can be extended using extension feature.",
+    )
+
+    parser.add_argument(
+        "-t",
+        "--test",
+        default=[],
+        action="append",
+        type=args_file,
+        help="Specify directory in which test will be searched. "
+        "If flag is not used then runner searches for tests in "
+        "phoenix-rtos-project directory. Flag can be used multiple times.",
+    )
+
+    parser.add_argument(
+        "-p",
+        "--port",
+        help="Specify serial to communicate with device board. "
+        "Default value depends on the target",
+    )
+
+    parser.add_argument(
+        "-b",
+        "--baudrate",
+        default=115200,
+        help="Specify the connection speed of serial. " "By default uses %(default)d",
+    )
+
+    parser.add_argument(
+        "--no-flash",
+        default=False,
+        action="store_true",
+        help="Board will not be flashed by runner.",
+    )
+
+    parser.add_argument(
+        "--no-test",
+        default=False,
+        action="store_true",
+        help="Tests will be not run by runner. This option may be helpful to only flash the device "
+        "or check if yamls are parsed correctly."
+    )
+
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        action="count",
+        default=0,
+        help="Increases verbosity level",
+    )
```

Comment:
i'd rather use well named constants than writing multiple v and hoping for the best

Reply:
Less readable though setting verbosity levels is, that may be more extensible solution.